docs/userguide/configuration.rst

```diff
@@ -598,6 +602,7 @@ Can be one of the following:
 .. _`CouchDB`: http://www.couchdb.com/
 .. _`Couchbase`: https://www.couchbase.com/
 .. _`Consul`: https://consul.io/
+.. _`AzureBlockBlob`: https://azure.microsoft.com/en-us/services/storage/
```

Comment:
Correct me if I am wrong, but I think I found a more accurate link [here](https://azure.microsoft.com/en-us/services/storage/blobs/).

Reply:
Done in e6284b0.